### PR TITLE
feat: Add pod blog

### LINKED ISSUE #5: Add pod stats
Add a way to display the number of contributions between fellows. Make a little leaderboard or similar to motivate people to add new issues!

### LINKED ISSUE #8: Add pod blog
It would be awesome to add a blog feature to this repository where fellows can share their insights and people can tune in!

### PR DESCRIPTION
**Pull request by ditsuke**
Closes #8. WIP

## Please, go through these steps before you submit a PR.

- [x] My Pod Leader knows I'm working on this Pull Request
- [x] I've explained what the Pull Request is adding.
- [x] I've explained why this is important.

### CODE REVIEW

**Comment by KhudaDad414** [comment]
@ditsuke hey, I think you link the wrong issue.

**Comment by ditsuke** [comment]
> @ditsuke hey, I think you link the wrong issue.

Whoops, fixed!

**Comment by harshsharma6401** [comment]
There are no blogs added yet. Need any help ?

**Comment by psahithireddy** [comment]
@harshsharma6401 we just made pages and displays, anyone can add blogs at any time (guess that was the goal). Currently, we made a template for 2 blogs, which we can easily extend,.@ditsuke I guess we will add instructions to add a blog in readme or maybe they can start an issue and tag us.

**Comment by harshsharma6401** [comment]
> @harshsharma6401 we just made pages and displays, anyone can add blogs at any time (guess that was the goal). Currently, we made a template for 2 blogs, which we can easily extend,.@ditsuke I guess we will add instructions to add a blog in readme or maybe they can start an issue and tag us.

Gotcha!

**Comment by ditsuke** [comment]
Ready for review!

File: _layouts/blog.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+                <a href="/">
+                    <img src="/assets/img/mlh-prep.png" style="width:100px" alt="mlh-prep-banner"/>
+                </a>
+            </div>
+            <div class="title" style="font-size: 40px;">
+                {{ page.title }}
+            </div>
+            <div class="title" style="font-size: 20px;">
+                {{page.author}}
+            </div>
```
Comment (yyjlincoln) [inline _layouts/blog.html:17]: You seem to be using the default layout & adding a custom banner here - which was what `projects.html` did. Can you try and use layout: page (which has the unified nav bar etc) to ensure consistent layout?

Not sure how you can deal with `{{page.author}}` though. Probably add a `{{ page.subtitle }}` in `_layouts/page.html` (and a if statement in HTML)? That way, pages without `subtitle` will also work as expected.

File: _layouts/default.html [hunk 2]
```diff
@@ -6,15 +6,15 @@
...
+    {% include page-transition.html %}
 
-{% include page-transition.html %}
+    {% include footer.html %}
 
-{% include footer.html %}
+    {% include toggle.html %}
+</body>
 
-{% include toggle.html %}
```
Comment (yyjlincoln) [inline _layouts/default.html:18]: Please don't change this file unnecessarily - it seems that this change will break a lot of pages.

Comment (yyjlincoln) [reply _layouts/default.html:18]: Is there a reason for this change?

File: index.html [hunk 3]
```diff
@@ -6,26 +6,30 @@
...
+        {% include map.html %}
 
-    {% include description.html %}
+        {% include toggle.html %}
 
-    {% include projects.html %}
+        {% include page-transition.html %}
 
-    {% include map.html %}
+    </main>
```
Comment (yyjlincoln) [inline index.html:27]: Can't remember the exact reason why we used `.main-content` (probably javascript-related issues?) but I believe we used it for a reason (#24). I do understand that `<main>` is cleaner than `.main-content` but let's keep it as `.main-content` for now & we'll start another PR to change it to `.main-content` later. Thanks.

**Review by yyjlincoln — CHANGES_REQUESTED** [review]
Great work! Please check the comments.

**Comment by yyjlincoln** [comment]
Everything else looks great!

File: _layouts/default.html (hunk 2 above)
Comment (ditsuke) [reply _layouts/default.html:18]: I believe this was for improving the layout semantics. Looking at it again, I believe these changes are beyond the scope of this PR. I'll be reverting the relevant commit and using it in another PR!

**Review by yyjlincoln — APPROVED** [review]
LGTM! Thanks a lot. We can start another PR to improve layout semantics.
It would be great if you can fix the banner issue as well but that's not a requirement - I'm fine with both. I thought it would look better.

**Comment by yyjlincoln** [comment]
Got my booster shot today so I'm really tired. I've also worked a bit too hard in the last week so I've decided to take a break on the weekends. Sorry that I couldn't help much today :(.

File: _layouts/blog.html (hunk 1 above)
Comment (ditsuke) [reply _layouts/blog.html:14]: We use now the navbar with 0c98638!

**Comment by ditsuke** [comment]
> Got my booster shot yesterday so I'm really tired. I've also worked a bit too hard in the last week so I decided to take a break on the weekends. Sorry that I couldn't help much today :(.

Thanks for approving Lincoln! Glad to hear you got the booster shot. You totally deserve the rest on weekends: you rocked and pulled more than your weight on weekdays and I definitely don't endorse working on weekends. Take care and hope you're good as new soon!

**Comment by yyjlincoln** [comment]
LGTM! We can merge it if we can get another approval

**Review by Dhruv-Sachdev1313 — APPROVED** [review]
Looks Great!!

**Comment by ditsuke** [comment]
2 approvals and passing checks - merging! thanks everyone

**Pull request merged**